.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,31 @@
+name: Bug
+description: File a bug report
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Before opening a bug report, please search for the behaviour in the existing issues. 
+        
+        ---
+        
+        Thank you for taking the time to file a bug report. To address this bug as fast as possible, we need some information.
+  - type: input
+    id: browser
+    attributes:
+      label: Broswer
+      description: "Which Web Browser do you use? Please provide the version as well."
+      placeholder: "Google Chrome"
+    validations:
+      required: true  
+  - type: textarea
```

Comment:
Cna you remove this field and instead have 2 fields - current behaviour and expected behaviour